serverless/README.md

```diff
@@ -1,90 +1,15 @@
-
-#### Include original headers
-Enable `Include original headers` when subscribing to the SNS topics from SES. To pass the message id to the lambda, we include the message id in the sent message, using the header `X-SMTPAPI`. 
-
-### Handling updates from Sendgrid
-We use Sendgrid as a backup email provider. Set the [Sendgrid webhook url](https://sendgrid.com/docs/for-developers/tracking-events/getting-started-event-webhook-security-features) to the api gateway endpoint that points to this lambda.
+See [eb-env-update-docs](eb-env-update/README.md) for details on `eb-env-update`
 
+TODO: Include README for `redaction-digest` and `database-backup`
 
-### Environment variables
-| Name                  | Description                                                                       |
-| --------------------- | --------------------------------------------------------------------------------- |
-| `DB_URI`              | URI to the postgres database                                                      |
-| `CALLBACK_SECRET`     | Secret for basic auth                                                             |
-| `MIN_HALT_NUMBER`     | Halt if there is this minimum number of invalid recipients, and it exceeds the percentage threshold |
-| `MIN_HALT_PERCENTAGE` | Halt if the percentage of invalid recipients exceeds this threshold. Supply a float from 0 to 1  |
-| `SENDGRID_PUBLIC_KEY` | Public key used to verify webhook events from sendgrid  |
-
-
-## SMS status update from Twilio
-
-`log-twilio-callback`handles SMS delivery callbacks from Twilio. 
-
-### Configuring callback url 
-Set the sms callback url when sending a message via the Twilio sdk or in the Twilio settings. Read [https://www.twilio.com/docs/sms/outbound-message-logging#receive-a-status-callback](https://www.twilio.com/docs/sms/outbound-message-logging#receive-a-status-callback)
-
-### Authorizing incoming Twilio requests
-The callback url provided includes username and password, where `username` is a random string and `password` is a hash of `username, messageId, campaignId and a secret token`. 
-
-[Postman API Gateway Authorizer](postman-api-gateway-authorizer) ensures that incoming Twilio requests have an Authorization header present. `log-twilio-callback` generates a password from the attributes as mentioned above, and compares it with the password in the Twilio Authorization header to authenticate the request.
-
-### Finalised delivery statuses
-`log-twilio-callback`is invoked for every change in status of a message. Only finalized statuses are processed and the rest are ignored. Read more about [Twilio Message Statuses](https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-).
+The serverless functions are deployed onto Lambda. For more details on how the serverless functions are set up and how they work, see [here](https://docs.google.com/document/d/1ZYvCKgQK5DhZAO1MkxF5le0nplHYVmi08MRWXia0UQw).
```

Comment:
We shouldn't replace README with Google Docs as these documents are private to OGP

Reply:
OK done